src/options.json

```diff
@@ -0,0 +1,58 @@
+{
+  "title": "install-webpack-plugin options",
+  "type": "object",
+  "definitions": {
+    "Dev": {
+      "anyOf": [
+        {
+          "type": "boolean"
+        },
+        {
+          "instanceof": "Function"
+        }
+      ],
+      "description": "Install as development dependencies"
+    },
+    "NPM": {
+      "type": "boolean",
+      "description": "Use NPM to install dependencies."
+    },
+    "PeerDependencies": {
+      "type": "boolean",
+      "description": "Install missing peer dependencies"
+    },
+    "Prompt": {
+      "type": "boolean",
+      "description": "Show a prompt to confirm installation."
+    },
+    "Quiet": {
+      "type": "boolean",
+      "description": " Reduce amount of console logging"
+    },
+    "Yarn": {
+      "type": "boolean",
+      "description": "Use yarn to install dependencies."
+    }
+  },
+  "additionalProperties": false,
+  "properties": {
+    "dev": {
+      "$ref": "#/definitions/Dev"
+    },
+    "npm": {
+      "$ref": "#/definitions/NPM"
+    },
+    "peerDependencies": {
+      "$ref": "#/definitions/PeerDependencies"
+    },
+    "prompt": {
+      "$ref": "#/definitions/Prompt"
+    },
+    "quiet": {
+      "$ref": "#/definitions/Quiet"
+    },
+    "yarn": {
+      "$ref": "#/definitions/Yarn"
+    }
+  }
+}
```

Comment:
The next step - remove `peerDependencies` and `dev` options in favor `dependecies: { dev: boolean, peer: boolean }`